modules/dop/endpoints/projectpipelinefiletree.go

```diff
@@ -159,8 +159,14 @@ func (e *Endpoints) DeleteFileTreeNode(ctx context.Context, r *http.Request, var
 		return apierrors.ErrDeleteFileTreeNode.NotLogin().ToResp(), nil
 	}
 
+	inode, err := url.QueryUnescape(vars["inode"])
+	if err != nil {
+		logrus.Errorf("QueryUnescape pipeline filetree inode %v failed", vars["inode"])
```

Comment:
if err not returned, Warnf is better. And why not return err, and fallback to plain var["inode"] ?

Reply:
if it is invalid URL escaper, treat it as ordinary inode